modules/exploits/windows/local/lexmark_driver_privesc.rb

```diff
@@ -74,44 +74,48 @@ def initialize(info = {})
 
   # Check to see if a there are Lexmark drivers in the driver store.
   # If there are, validate that they are similar to the ones we want
-  # to exploit.
+  # to exploit. The user can specify the driver they'd like to exploit
+  # as option. Otherwise, the first driver is selected.
```

Comment:
`Otherwise the first vulnerable Lexmark driver listed in the driver store will be selected.`